### PR TITLE
init gm

### PR DESCRIPTION
**Pull request by HaoXuan40404**
add guomi

### CODE REVIEW

File: libdevcore/GmHash.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/*
+        This file is part of solidity.
```
Comment (wheatli) [inline libdevcore/GmHash.cpp]: FISCO-BCOS

File: libdevcore/GmHash.cpp [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        solidity is distributed in the hope that it will be useful,
+        but WITHOUT ANY WARRANTY; without even the implied warranty of
+        MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+        GNU General Public License for more details.
+
+        You should have received a copy of the GNU General Public License
+        along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file SHA3.cpp
+ * @author Gav Wood <i@gavwood.com> asherli
```
Comment (wheatli) [inline libdevcore/GmHash.cpp]: your name plz

File: libdevcore/GmHash.cpp (hunk 1 above)
Comment (HaoXuan40404) [reply libdevcore/GmHash.cpp]: done

File: libdevcore/GmHash.cpp (hunk 2 above)
Comment (HaoXuan40404) [reply libdevcore/GmHash.cpp]: done

**Review by wheatli — APPROVED** [review]

**Pull request merged**